core/dbt/flags.py

```diff
@@ -72,6 +75,12 @@ def set_from_args(args):
     PARTIAL_PARSE = getattr(args, 'partial_parse', None)
     MP_CONTEXT = _get_context()
 
+    USE_COLORS = getattr(args, 'use_colors', USE_COLORS)
+    if USE_COLORS:
+        ui.use_colors()
+    else:
+        ui.do_not_use_colors()
```

Comment:
With my suggested change of having flags.USE_COLORS as the source of truth, we can get rid of the if-statement here and just set USE_COLORS.